book/src/zcash/ywallet-demo.md

```diff
@@ -169,9 +187,31 @@ If you prefer to pass the randomizer as a file by using the `--randomizer`
 argument, you will need to convert it to binary format.
 ```
 
-The protocol should run and complete succesfully. You should still be at the
-Coordinator CLI. It has just printed the final FROST-generated signature.
-Hurrah! Copy it (just the hex value).
+### Participant 1 (alice)
+
+In the third terminal, Participant 1, run the following:
+
+```
+export PW=foobar10
+cargo run --bin participant -- -C redpallas --http --key-package key-package-1.json -u alice -w PW
+```
+
+(We are using "alice" again. There's nothing stopping a Coordinator from being a
+Partcipant too!)
+
+### Participant 2 (bob)
+
+In the fourth terminal, for Participant 2, run the following:
+
+```
+cargo run --bin participant -- -C redpallas --http --key-package key-package-2.json -u bob -w PW
```

Comment:
Here it prompted me to set the password. Should we add `export PW=foobar10`  to this command also or will it depend on how people have configured things in their terminals?

Reply:
Good catch, fixed in 412dd015